pkg/console/subresource/configmap/configmap.go

```diff
@@ -119,7 +123,7 @@ func Stub() *corev1.ConfigMap {
 	return configMap
 }
 
-func NewYamlConfig(host string, logoutRedirect string, brand operatorv1.Brand, docURL string, apiServerURL string) []byte {
+func NewYamlConfig(host string, logoutRedirect string, brand operatorv1.Brand, docURL string, apiServerURL string, customProductName string, customLogoFile string) []byte {
```

Comment:
nit: This, however, should be generic.  `productName`, `logoFile`.  This `func` has no idea if its custom or default.